pandas/io/formats/style.py

```diff
@@ -2784,31 +2784,33 @@ def background_gradient(
 
         Shading the values column-wise, with ``axis=0``, preselecting numeric columns
 
-        >>> df.style.{name}_gradient(axis=0)  # doctest: +SKIP
+        >>> df.style.background_gradient(axis=0)  # doctest: +SKIP
 
         .. figure:: ../../_static/style/{image_prefix}_ax0.png
 
         Shading all values collectively using ``axis=None``
 
-        >>> df.style.{name}_gradient(axis=None)  # doctest: +SKIP
+        >>> df.style.background_gradient(axis=None)  # doctest: +SKIP
 
         .. figure:: ../../_static/style/{image_prefix}_axNone.png
 
         Compress the color map from the both ``low`` and ``high`` ends
 
-        >>> df.style.{name}_gradient(axis=None, low=0.75, high=1.0)  # doctest: +SKIP
+        >>> df.style.background_gradient(axis=None,
+        ...                             low=0.75, high=1.0)  # doctest: +SKIP
```

Comment:
I think you need to indent `low` with an extra space. Also, I'd probably move `high` to a different line too.